src/main/java/org/openlmis/fulfillment/web/OrderController.java

```diff
@@ -362,12 +357,11 @@ private Order createSingleOrder(OrderDto orderDto,
 
     LOGGER.debug("Creating new order");
 
-    ProgramDto program = programReferenceDataService.findOne(order.getProgramId());
-    OrderNumberConfiguration orderNumberConfiguration =
-        orderNumberConfigurationRepository.findAll().iterator().next();
+    OrderCodeGenerator orderCodeGenerator = extensionManager.getExtension(
```

Comment:
Please add integration tests for using extension point here.

Reply:
Done